### PR TITLE
Added GraphQL bridge to REST API

### PR DESCRIPTION
**Pull request by dacz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -298,7 +298,6 @@ A collection of awesome things regarding React ecosystem.
 ##### React Projects
 * [gatsby - Transform plain text into dynamic blogs and websites using React.js](https://github.com/gatsbyjs/gatsby)
 * [react-blessed - A react renderer for blessed terminal interface library](https://github.com/Yomguithereal/react-blessed)
-* [MJML - The only framework that makes responsive-email easy. Based on React](https://mjml.io)
```
Comment (paulmelnikow) [inline README.md]: Why is this removed?

File: README.md [hunk 2]
```diff
@@ -510,7 +509,6 @@ A collection of awesome things regarding React ecosystem.
 * [react-responsive - Media queries in react for responsive design](https://github.com/wearefractal/react-responsive)
 * [react-cursor - Functional state management abstraction for use with Facebook React](https://github.com/dustingetz/react-cursor)
 * [Omniscient.js - Abstracting React components for fast top down render of immutable data](https://github.com/omniscientjs/omniscient)
-* [Touchstonejs - React.js powered UI framework for developing beautiful hybrid mobile apps.](http://touchstonejs.io/)
```
Comment (paulmelnikow) [inline README.md]: Instead of removing this, how about replacing it with the working link?

https://github.com/touchstonejs/touchstonejs

File: README.md (hunk 1 above)
Comment (dacz) [reply README.md]: Not sure. I had a failing tests because some resources were not accessible. I see it's working now so I add it back.
